sorc/checkout.sh

```diff
@@ -177,6 +177,13 @@ for checkout_pid in $(jobs -p); do
   wait "${checkout_pid}" || errs=$((errs + $?))
 done
 
+# Temporary hack to check out a UPP verison that works on Orion
+# This can be removed once the UFS UPP version advances to or beyond 78f369b
+cd "${topdir}/ufs_model.fd/FV3/upp" || exit 1
+git checkout 78f369b
+cd "${topdir}" || exit 1
+# End hack
+
```

Comment:
What happens to the ufs-weather-model build if you use this UPP hash.
A better solution would be to checkout the orion module file for this hash alone
something like:
```
cd "${topdir}/ufs_model.fd/FV3/upp"
git checkout 78f369b modulefile_for_orion
```

Reply:
I built the whole system with this hack and UFS builds fine. I can try a more targeted option but:
- it will be more than one file, because UPP has moved to a machine + common approach like UFS
- I am worried about API mismatches if the module versions have changed (though those should be evident during a build)